portal/models/questionnaire_bank.py

```diff
@@ -321,22 +325,25 @@ def calculated_start(self, trigger_date, as_of_date=None):
                 (relative_start, ic) = recurrence.active_interval_start(
                     trigger_date=trigger_date, as_of_date=as_of_date)
                 if relative_start:
-                    return (relative_start, ic)
-            return (None, None)  # no active recurrence
+                    return QBD(relative_start=relative_start, iteration=ic,
+                               recur=recurrence, questionnaire_bank=None)
```

Comment:
Why don't we populate QBD.questionnaire_bank here?  isn't it `self`?

Reply:
Good point, I'll fix that